Dockerfile_cli

```diff
@@ -37,6 +37,9 @@ COPY *.py /usr/local/bin/simdem_cli/
 RUN chmod +x /usr/local/bin/simdem_cli/main.py
 RUN ln -s /usr/local/bin/simdem_cli/main.py /usr/local/bin/simdem
 
+# Require commands for demos
```

Comment:
There is a very lets set of requirements demo's might have. We should not be including these in the SimDem container, it will grow exponentially.

The correct way to handle this is in the demo script itself, such as in a preparation steps. If you don't want it in the actual demo then you can always put it in a [pre-requisites step](https://github.com/Azure/simdem/blob/master/demo_scripts/simdem/prerequisites/script.md)